common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java

```diff
@@ -145,7 +145,15 @@ public void onReceivedError(final WebView view,
     public void onReceivedError(@NonNull final WebView view,
```

Comment:
I think it's worth adding some javadoc here to explain what's going on... you may want to link to the android documentation as well.

Reply:
ACK

Reply:
77225978b2c3236b3a18051f4de76f040fdfeb4e